DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -177,6 +177,20 @@ namespace pat {
     void scaleEnergy(double fScale) override { scaleEnergy(fScale, "Unscaled"); }
     void scaleEnergy(double fScale, const std::string& level);
 
+    // Methods to work with a JER correction factor
+    //
+    // is the JER correction factor valid, i.e. has it been written using the saveJerFactor method
+    bool isJerFactorValid() const;
+
+    // load the currently saved JER correction factor and check whether the factor was propely set before loading
+    float loadJerFactor() const;
```

Comment:
```suggestion
    float jerFactor() const;
```
may be clearer, also follows better the rule 2.11: http://cms-sw.github.io/cms_coding_rules.html

> For a getter method, using the value name is preferred, e.g. momentum().

Reply:
done